Library/Input_Configuration/OPTIONS_GeneralConfiguration.m

```diff
@@ -183,7 +145,29 @@
     else
         str = [ 'Error: User selected FacSafe: ', num2str(OPTIONS.FacSafe), '. FacSafe must be >= 0.' ];
         error(str);
-    end    
+    end
+    
+    % FacSafe[High, Low]
+    if ( OPTIONS.FacSafeHigh == 0 )
```

Comment:
Additional issue here is if the user sets FacSafeHigh to be negative or NaN. That should be checked for as well.